### ISSUE TITLE
glx2 does not always replace SE when opening a script

### ISSUE THREAD

**Issue by macMikey** [mwieder/glx2ScriptEditor#8]
example:
I just did a find for a function name (in the card editor), and double-clicked on one of the entries.
the SE opened, instead of glx2

**Comment by mwieder** [comment]
Again, what's a 'card editor'?

**Comment by macMikey** [comment]
For lack of a better word, it's when I am not in the script editor.  What did we used to call it in HC?

**Comment by mwieder** [comment]
Ah. I don't remember what HC called that thing. You're referring to the "Find and Replace" palette? If so, that seems to be stack "revSearch". And double-clicking an entry brings up glx2 for me. I'm assuming you've grabbed the latest version here. I realize saying "it works for me" isn't much help. Is there some other plugin that's getting in the way? What *should* be happening is that the "editScript" message is intercepted by the glx2 script editor frontscript. If something else is getting it first then that might explain things.

**Comment by macMikey** [comment]
No, that's not what I meant. I meant whatever it was called when you had either the button tool or the field tool selected, and therefore you were editing buttons or fields. Now, we have the, uh, pointer and the pointer with the compass. I think of that as the "card editor", because you're not in the script editor, you're editing cards (or backgrounds, etc.)
I have Seth installed, too (but it seems to slow down the SE so I was giving glx2 another try). Maybe Seth is breaking the behavior. I'll have to try that tomorrow.

**Comment by mwieder** [comment]
OK - I'm *really* confused now. So I select an object with the cursor in... er... pointer and compass mode... and I press control-F to find things. And that brings up the Find and Replace palette. If I search for a handler I get an entry, and then double-clicking it brings up the proper script editor.

Seth could well be a conflict if it's trying to do similar things in a frontscript. I'll keep my fingers crossed that it's the issue because I'm not sure where else to look.

**Comment by mwieder** [comment]
Just checked... yep... Seth also grabs the editScript message in a frontscript before glx2 has a chance to see it. But it does pass the message on when it's done colorizing, so I'm not sure why glx2 as a second frontscript isn't getting the message for you.

**Comment by macMikey** [comment]
So far, so good. I just disabled seth and both of the issues I reported stopped. I'm going to close this unless it comes back.

**Comment by mwieder** [comment]
Reopening to keep this flagged so I can see if there's a way to avoid the conflict.

**Comment by mwieder** [comment]
OK - give it a try with the new build and see if the conflict is gone.
